Kernel/Ordering.cpp

```diff
@@ -68,14 +70,23 @@ OrderingSP Ordering::s_globalOrdering;
  */
 bool Ordering::trySetGlobalOrdering(OrderingSP ordering)
 {
-
   if(s_globalOrdering) {
     return false;
   }
   s_globalOrdering = ordering;
   return true;
 }
 
+bool Ordering::unsetGlobalOrdering()
```

Comment:
I have written some code that relies on there being a global ordering at some point. Should I not have?